doc/xml/user-guide.xml

```diff
@@ -2395,7 +2395,9 @@
 
         <p>Note that the <br-option>repo</br-option> option is not required when only <br-option>repo1</br-option> is configured in order to maintain backward compatibility. However, the <br-option>repo</br-option> option <i>is</i> required when a single repo is configured as, e.g. <br-option>repo2</br-option>. This is to prevent command breakage if a new repository is added later.</p>
 
-        <p>The <cmd>archive-push</cmd> command will always push WAL to the archive in all configured repositories but backups will need to be scheduled individually for each repository. In many cases this is desirable since backup types and retention will vary by repository. Likewise, restores must specify a repository. It is generally better to specify a repository for restores that has low latency/cost even if that means more recovery time. Only restore testing can determine which repository will be most efficient.</p>
+        <p>The <cmd>archive-push</cmd> command will always push WAL to the archive in all configured repositories. When a repository cannot be reached, WAL will still be pushed to other repositories. However, for this to be work effectively <br-setting>archive-aync=y</br-setting> must be enabled, otherwise the other repositories can only get one WAL segment ahead of the unreachable repository. Also note that if WAL cannot be pushed to any repository then <postgres/> will not remove it from the <path>pg_wal</path> directory, which can cause the volume to run out of space.</p>
```

Comment:
Hi,

I would suggest these little readability changes:
> However, for this to work effectively, <br-setting>archive-aync=y</br-setting> must be enabled; otherwise, the other repositories can only get one WAL segment ahead of the unreachable repository. Also, note that if WAL cannot be pushed to any repository, then <postgres/> will not remove it from the <path>pg_wal</path> directory, which can cause the volume to run out of space.

Otherwise this looks good to me :+1:

Reply:
This looks good. Updated in f63337b7a with minor edits, including bringing back the word PostgreSQL.

Reply:
Perfect :ok_hand: